docs/event-serialization.md

```diff
@@ -97,13 +97,18 @@ KAFKA_PORT_PLAIN=17092 KAFKA_PORT_SSL=17093 HOST=localhost docker-compose -f int
 # 2. Start Rig
 # Here we say to use Avro, consume on topic "rigRequest" and use "rigRequest-value" schema from Kafka Schema Registry
 # Proxy is turned on to be able to produce Kafka event with headers (needed for cloud events)
-KAFKA_BROKERS=localhost:17092 \
-KAFKA_SERIALIZER=avro \
-KAFKA_SOURCE_TOPICS=rigRequest \
-PROXY_CONFIG_FILE=proxy/proxy.test.json \
-PROXY_KAFKA_REQUEST_TOPIC=rigRequest \
-PROXY_KAFKA_REQUEST_AVRO=rigRequest-value \
-mix phx.server
+docker run -d --name rig \
+-e KAFKA_BROKERS=kafka:9292 \
+-e KAFKA_SERIALIZER=avro \
+-e KAFKA_SCHEMA_REGISTRY_HOST=kafka-schema-registry:8081 \
+-e KAFKA_SOURCE_TOPICS=rigRequest \
+-e PROXY_CONFIG_FILE=proxy/proxy.test.json \
+-e PROXY_KAFKA_REQUEST_TOPIC=rigRequest \
+-e PROXY_KAFKA_REQUEST_AVRO=rigRequest-value \
+-e LOG_LEVEL=debug \
+-p 4000:4000 -p 4010:4010 \
+--network kafka_tests_default \
+accenture/reactive-interaction-gateway
```

Comment:
works okay, however, below there's this:

> In terminal you should see something like below

and I don't see this, partly because docker runs the container in detached mode, but even with `docker logs -f rig` I don't see the line as described. Does this mean it doesn't work as expected?

Reply:
Are you sure? Just tested it and I can see:

```
11:05:48.880 [debug] Decoded Avro message="{\"example\":\"test\"}"
application=rig_kafka module=RigKafka.Avro function=decode/2 file=lib/rig_kafka/avro.ex line=29 pid=<0.1935.0>

11:05:48.882 [debug] [:start_object, {:string, "contenttype"}, :colon, {:string, "avro/binary"}, :comma, {:string, "data"}, :colon, :start_object, {:string, "example"}, :colon, {:string, "test"}, :end_object, :comma, {:string, "id"}, :colon, {:string, "069711bf-3946-4661-984f-c667657b8d85"}, :comma, {:string, "rig"}, :colon, :start_object, {:string, "correlation"}, :colon, {:string, "g2dkAA1yaWdAMTI3LjAuMC4xAAAI7AAAAAAB"}, :comma, {:string, "headers"}, :colon, :start_array, :start_array, {:string, "accept"}, :comma, {:string, "*/*"}, :end_array, :comma, :start_array, {:string, "content-length"}, :comma, {:string, "221"}, :end_array, :comma, :start_array, {:string, "content-type"}, :comma, {:string, "application/json"}, :end_array, :comma, :start_array, {:string, "host"}, :comma, {:string, ...}, :end_array, ...]
application=rig module=Rig.EventStream.KafkaToFilter function=kafka_handler/1 file=lib/rig/event_stream/kafka_to_filter.ex line=20 pid=<0.1935.0>
```